test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -634,24 +638,41 @@ public void RendersLinkTagsForGlobbedHrefResults()
             // Assert
             Assert.Equal("link", output.TagName);
             Assert.Equal("/css/site.css", output.Attributes["href"].Value);
-            Assert.Equal("<link rel=\"stylesheet\" href=\"HtmlEncode[[/base.css]]\" />", output.PostElement.GetContent());
+            var content = HtmlContentUtilities.HtmlContentToString(output, new HtmlTestEncoder());
+            Assert.Equal(expectedContent, content);
         }
 
         [Fact]
-        public void RendersLinkTagsForGlobbedHrefResults_UsingProvidedEncoder()
```

Comment:
This test was a dupe of `RendersLinkTagsForGlobbedHrefResults()`. Fairly different now. Same deal for `RendersScriptTagsForGlobbedSrcResults_UsesProvidedEncoder()` and `RendersScriptTagsForGlobbedSrcResults()`.